Package.swift

```diff
@@ -12,12 +12,12 @@ let package = Package(
     targets: [
         .binaryTarget(
                     name: "Sentry",
-                    url: "https://github.com/getsentry/sentry-cocoa/releases/download/8.49.2/Sentry.xcframework.zip",
+                    url: "https://github.com/getsentry/sentry-cocoa/releases/download/8.49.3/Sentry.xcframework.zip",
                     checksum: "85cf16b4fa2dd0ae58b1e5c62fd8c829c94e5dc92251c3a1de8e71b3fed195ef" //Sentry-Static
```

Comment:
`h`: If the URL to the zip changes, the checksum must also change. Otherwise, we could break SPM, when compiling from the main branch. Anyways this should be 8.50.0, and we shouldn't update this manually. Only the release workflow should do that.